src/main/java/com/puppycrawl/tools/checkstyle/xpath/ElementNode.java

```diff
@@ -55,6 +57,9 @@
     /** Represents text of the DetailAST. */
     private final String text;
 
+    /** Represents text of the DetailAST. */
+    private final int positionAmongSiblings;
```

Comment:
this was not done.

from my comment above:
>It is better to name field as index indexAmongSiblings, as position word does not imply strict sequence.

Reply:
sorry, done